page-home.php

```diff
@@ -0,0 +1,194 @@
+<?php
```

Comment:
franking the confusion between `home.php` and `front-page.php` always gets me. So as this is a page being set at the front page in the reading settings (with a News page being set for posts) this makes sense to me